jgiven-core/src/test/java/com/tngtech/jgiven/GivenTestStep.java

```diff
@@ -132,12 +147,72 @@ public GivenTestStep table_as_parameter(@Table String[] params) {
         return self();
     }
 
-    @As( "another description" )
     @IntroWord
+    @As( "another description" )
     public GivenTestStep an_intro_word_with_an_as_annotation() {
         return self();
     }
 
+    @FillerWord
```

Comment:
what happens if someone has the genius idea of stacking `@FillerWord` and `@IntroWord`? We should make sure that that behavior is well defined and known.

Reply:
In this situation '@FillerWord' will be ignored. I've added clarification of this to the docs.